app/controllers/orga/projects_controller.rb

```diff
@@ -14,6 +14,15 @@ def accept
     redirect_to [:orga, :projects]
   end
 
+  def pending
```

Comment:
Shall we change the name of the controller action to correspond to the acts-as-state-machine state, i.e. `start_review`?  I don't mind either option so I let you decide.